interface/interface.dm

```diff
@@ -52,7 +52,15 @@
 	set desc = "Report an issue"
 	set hidden = 1
 	if(config.githuburl)
-		if(alert("This will open the Github issue reporter in your browser. Are you sure?",,"Yes","No")=="No")
+		var/message = "This will open the Github issue reporter in your browser"
+		var/first = TRUE
+		for(var/line in revdata.testmerge)
+			if(line)
+				if(first)
+					first = FALSE
+					message += ". Do note the following PR testmerges are active and bugs caused by them should be reported in their respective threads, not the issue tracker:"	

```

Comment:
"The following experimental changes are active and are probably the cause of any new or sudden issues you may experience.
[list.Join()]
If applicable, please post issues relating to experimental changes in the thread for that change."